toolchain/sem_ir/inst_kind.cpp

```diff
@@ -15,17 +15,16 @@ CARBON_DEFINE_ENUM_CLASS_NAMES(InstKind) = {
 
 auto InstKind::definition_info(InstKind inst_kind) -> const DefinitionInfo& {
   static constexpr InstKind::DefinitionInfo DefinitionInfos[] = {
-#define CARBON_SEM_IR_INST_KIND(Name) SemIR::Name::Kind.info_,
+#define CARBON_SEM_IR_INST_KIND(Name) Name::Kind.info_,
 #include "toolchain/sem_ir/inst_kind.def"
   };
   return DefinitionInfos[inst_kind.AsInt()];
 }
 
 auto InstKind::value_kind() const -> InstValueKind {
   static constexpr InstValueKind Table[] = {
-#define CARBON_SEM_IR_INST_KIND(Name)                           \
-  Internal::HasTypeIdMember<SemIR::Name> ? InstValueKind::Typed \
-                                         : InstValueKind::None,
+#define CARBON_SEM_IR_INST_KIND(Name) \
+  Internal::HasTypeIdMember<Name> ? InstValueKind::Typed : InstValueKind::None,
```

Comment:
The namespace qualification in this case is necessary: the enum-like type `InstKind` has members whose names are the same as the typed instructions, and the `SemIR::` disambiguates. (This PR doesn't build because of this.)

Reply:
Yeahhhh, I'd thought I had this building but apparently not.